docs/README.md

```diff
@@ -1,114 +1,232 @@
 # ScalarDB Benchmarks
 
-This repository contains benchmark programs for ScalarDB.
+This tutorial describes how to set up and run benchmarking tools for ScalarDB.
 
 ## Available workloads
```

Comment:
Should this heading be as follows instead?:

```suggestion
## Available benchmarks
```

Reply:
How about `benchmark workloads`? It seems a bit more clearer to me.

Reply:
That sounds good! Thank you!